conans/test/unittests/client/generators/premake_test.py

```diff
@@ -47,8 +47,8 @@ class PremakeGeneratorTest(unittest.TestCase):
     conan_libdirs_MyPkg2 = {{"{lib2}"}}
     conan_bindirs_MyPkg2 = {{"{bin2}"}}
     conan_libs_MyPkg2 = {{"libbar"}}
-    conan_cppdefines_MyPkg2 = {{"MYDEFINE2"}}
-    conan_cppflags_MyPkg2 = {{"-march=native"}}
+    conan_defines_MyPkg2 = {{"MYDEFINE2"}}
+    conan_cxxflags_MyPkg2 = {{"-march=native"}}
```

Comment:
```suggestion
    conan_cppflags_MyPkg2 = {{"-march=native"}}
```

Reply:
Finally we agreed on having ``cpp_info.cppflags`` converted to ``conan_cxxflags``. So no further action is required.